client/components/SimilarPackageCard/SimilarPackageCard.js

```diff
@@ -46,10 +46,10 @@ export default class SimilarPackageCard extends Component {
         return (
           <div>
             <div className="similar-package-card__number">
-              {Math.abs(comparisonSizePercent).toFixed(0) / 100} <span className="similar-package-card__shrink">×</span>
+              {1 + Math.abs(comparisonSizePercent).toFixed(0) / 100} <span className="similar-package-card__shrink">×</span>
             </div>
             <div className="similar-package-card__label">
-              {comparisonSizePercent > 0 ? 'Larger' : 'Smaller'}
```

Comment:
Instead of removing this, I think the `else if` should use the absolute value of `comparisonSizePercent`. The goal is to show the difference in terms of `<times> x` when it is more than 100% in either direction.

Reply:
Fixed